publisher.rb

```diff
@@ -0,0 +1,47 @@
+require 'aws-sdk-sqs'
+require 'byebug'
+require_relative 'load_aws'
+
+class Publisher
+  # Note: You can store the values of the aws access keys and region list in a config file (json) 
+  # from where you can get the values into the constants.
+  ACCESS_KEY_ID = AWS["access_key_id"]
+  SECRET_ACCESS_KEY = AWS["secret_access_key"]
+  REGION = AWS["region"]
+
+  attr_accessor :sqs
```

Comment:
Why attr_accessor ?

Reply:
Thanks for pointing this out. Will change it to attr_reader.